### ISSUE TITLE
@playwright/test & vitest browser mode aren't happy together! 

### ISSUE THREAD

**Issue by Bnaya** [vitest-dev/vitest#3807]
### Describe the bug

Since  https://github.com/microsoft/playwright/blob/207086180781e1daee52bcba0aefe3bed1b87b2e/docs/src/release-notes-js.md?plain=1#L167-L170, `@playwright/test` requires that the plain playwright package **won't** be installed.

Vitest browser mode, for playwright driver, the plain playwright package must be installed.
https://github.com/vitest-dev/vitest/blob/8693449b412743f20a63fd9bfa1a9054aa74613f/packages/vitest/src/node/browser/playwright.ts#L31-L33

Thus preventing using this tools on the same project.

I've opened a similar issue on the playwright repo:
https://github.com/microsoft/playwright/issues/24385

### Reproduction

N/A

### System Info

```shell
N/A
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
Should we check for installed `@playwight/test`, @Aslemammad ?

**Comment by Aslemammad** [comment]
We can check, but @Bnaya your aim is to use both these tools at the same time? I'm just trying to understand.

**Comment by Bnaya** [comment]
Yes, We use playwright test for e2e and vitest-browser for unit test

**Comment by Bnaya** [comment]
I think that playwright are mistaken for erroring if "playwright" is installed,
As it's a valid use case to use playwright as a library and playwright/test on the same project, even unrelated to vite

But it would be great to have a solution/workaround also from vitest side
